alliance/apps/backlog/views/github_webhook_view.py

```diff
@@ -31,6 +31,7 @@ def post(self, request):
         signature_ver = self.verify_signature(request)
         if not signature_ver:
             event = request.META.get(self.meta_key_formatter('X-GitHub-Event'))
+            response = None
```

Comment:
For example, I removed this line in a previous pull request. It's redundant because `response` gets defined in every code path below.